src/main/java/ch/njol/skript/expressions/ExprTarget.java

```diff
@@ -61,23 +61,28 @@
 	"delete targeted entity of player # for players it will delete the target",
 	"delete target of last spawned zombie # for entities it will make them target-less"
 })
-@Since("1.4.2, 2.7 (Reset)")
+@RequiredPlugins("Paper 1.19+ (enhanced raytracing)")
+@Since("1.4.2, 2.7 (Reset), INSERT VERSION (ignore blocks)")
 public class ExprTarget extends PropertyExpression<LivingEntity, Entity> {
 
+	private static final boolean PAPER_RAYTRACE = Skript.methodExists(LivingEntity.class, "rayTraceEntities", int.class, boolean.class);
+
 	static {
 		Skript.registerExpression(ExprTarget.class, Entity.class, ExpressionType.PROPERTY,
-				"[the] target[[ed] %-*entitydata%] [of %livingentities%]",
-				"%livingentities%'[s] target[[ed] %-*entitydata%]");
+				"[the] target[[ed] %-*entitydata%] [of %livingentities%]" + (PAPER_RAYTRACE ? " [ignoring :blocks]" : ""),
+				"%livingentities%'[s] target[[ed] %-*entitydata%]" + (PAPER_RAYTRACE ? " [ignoring :blocks]" : ""));
```

Comment:
This pattern will match `target entity blocks`

Reply:
ignoring blocks is one optional

Reply:
I'm saying `target entity blocks` doesn't make any sense. Fix the pattern

Reply:
You still can't do `target entity blocks` tho Tud, it's pattern is `[ignoring blocks]` not `[ignoring] [blocks]`

Reply:
It's `[ignoring :blocks]` `ignoring ` is a parse tag here, leaving the pattern with only `blocks`

Reply:
Oh lol, I see how you got confused on that. I adjusted it to make it easier to understand.